### PR TITLE
docs: explained how to see dispatched actions

### PR DESCRIPTION
**Pull request by dsebastien**
When using MockStore for test, it is useful to inspect the dispatched actions.
The initial tip indicated that it was possible but did not say how to.

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/ngrx/platform/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Documentation has been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?

Closes #

## Does this PR introduce a breaking change?

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: projects/ngrx.io/content/guide/store/testing.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ You can write tests validating behaviors corresponding to the specific state sna
 
 <div class="alert is-helpful">
 
-**Note:** All dispatched actions don't affect the state, but you can see them in the `Actions` stream.
+**Note:** All dispatched actions don't affect the state, but you can see them in the `Actions` stream. You can subscribe to it using `.scannedActions$.subscribe(...)` on the MockStore instance.
```
Comment (timdeschryver) [inline projects/ngrx.io/content/guide/store/testing.md]:
```suggestion
**Note:** All dispatched actions don't affect the state, but you can see them in the `Actions` stream. You can subscribe to the stream with the `scannedActions$` property on the `MockStore` instance, for example `store.scannedActions$.subscribe(action => console.log(action))`.
```

Comment (timdeschryver) [reply projects/ngrx.io/content/guide/store/testing.md]: What about the above?

**Comment by brandonroberts** [comment]
@dsebastien are you still planning on making these changes?

**Comment by brandonroberts** [comment]
Just reopen this PR when you're ready to pick it back up. Thanks!

**Pull request closed without merging**